src/atom.cpp

```diff
@@ -1251,7 +1250,6 @@ void Atom::data_atoms(int n, char *buf, tagint id_offset, tagint mol_offset,
               error->one(FLERR, "Invalid atom type {} in {}: {}", itype, location,
                          utils::trim(buf));
             type[nlocal - 1] = itype;
-            if (labelflag) type[nlocal - 1] = ilabel[itype - 1];
```

Comment:
I am confused about many of the changes in this PR. Simply removing lines like these essentially reverts the type label functionality within a data file, and eliminates one of the main benefits of the framework, which is to automatically sync types between multiple input files when using type labels.

Reply:
Please provide an example illustrating this.

Reply:
Please see attached for the example I was thinking of.
[in.zip](https://github.com/user-attachments/files/16517160/in.zip)